test/tests/api/v1_1/__init__.py

```diff
@@ -16,5 +17,6 @@
     'profiles.tests'
     'config.tests',
     'workflowTasks.tests',
-    'workflows.tests'
+    'workflows.tests',
+    'auth.tests'
```

Comment:
You mean add something like below in https://github.com/RackHD/RackHD/blob/master/test/run.py
```python
register(groups=['api-v1.1-withauth'], depends_on_groups=api_1_1.tests_withauth)
```
But the problem is whether this was enabled by default. If the auth disabled in config file. This tests withauth will still fail.